### ISSUE TITLE
warning occured 

### ISSUE THREAD

**Issue by DmrS5** [nprint/nprintml#88]
### What's up?

There's a warning "could not open X display" occured as shown in the figure.
I'm not sure it is because I'm running on VM.

What;s more,"No protocol specified"  appeared even  though I run with `-t -u` ,what should I do?

![2023-03-01_202135](https://user-images.githubusercontent.com/126075324/222140699-d71cb633-e33c-4a80-b0a0-11925af6e8e7.jpg)

### Due diligence

- [X] I have read the docs

**Comment by DmrS5** [comment]
> Was this resolved?

Not really, but I guess there's something wrong with my VM. I try to run nprintml on another device and it works.

**Comment by JordanHolland** [comment]
Interesting. What OS?

The X display warning could be an X11 forwarding issue and is most likely external to nPrintML. I'm more concerned with the no protocol specified error
